packages/machine/src/middleware/protocol-operation/op-generator.ts

```diff
@@ -41,15 +33,15 @@ export class EthOpGenerator extends OpGenerator {
         context,
         node,
         proposedState.state,
-        proposedState.cfAddr
+        proposedState.cfAddr!
```

Comment:
Why add `!`?

Reply:
now that `proposedState` has a proper type (it used to be of type `any`), tsc infers that `cfAddr` is possibly undefined, but this parameter does not accept undefined

Reply:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll If `tsc` is inferring that, there's a type somewhere along the way that can be better defined. Maybe we should check that instead of muting the compiler :(

Reply:
I can try to see if it is simple to fix, but I don't think this should block merging this, because the fact that cfAddr is not constrained by the type system to be not undefined is true before this PR as well